packages/payment-integration-api/src/common/utility/object-with-kebab-keys.ts

```diff
@@ -0,0 +1,13 @@
+import { kebabCase } from 'lodash';
+
+export default function objectWithKebabKeys(object: { [key: string]: any }) {
```

Comment:
Do we need this code in separate util? Is it in use more than 1 time?

And if we still need this helper I would recommend to change the naming to something like `objectWithKebabCaseKeys`💅

Reply:
Yes, we might use this method in braintree and paypal commerce package